examples/pachyderm-cd4ml/Makefile

```diff
@@ -0,0 +1,14 @@
+build:
+	make -C RSIServing build
+	make -C RSITraining build
+	make -C RSIDeployer build
+
+push: build
+	make -C RSIServing push
+	make -C RSITraining push
+	make -C RSIDeployer push
+
+kind_load: build
```

Comment:
We would be nice to include `KIND_NAME` that defaults to `kind` as in previous example.